src/middlewares/autosaveMiddleware.ts

```diff
@@ -43,27 +84,64 @@ const saveFormMiddleware = ({ getState, dispatch }: MiddlewareAPI<Dispatch<AnyAc
             }
 
             /**
-             * Default save operation CRUD
-             *
+             * final condition
              */
-
-            if (selectedCell
-                && (
-                    needToSaveTableChanges
-                    || (!isSelectTableCellInit && !isSendOperation)
-                    || (isSelectTableCellInit &&
-                        (selectedCell.widgetName !== action.payload.widgetName || selectedCell.rowId !== action.payload.rowId)
-                    )
+            const isNeedSaveCondition = isNotSaveAction &&
```

Comment:
The `sendOperation.associate` action from `actionGroup` to another BC causes autosave `OperationTypeCrud.save`. This should not happen because this operation calls the association widget popup. When performing an autosave in this case, the backend can respond with a `postInvoke` action to the autosave operation and the application will break.

Reply:
@AndrewRAM33 a mechanism of middleware overriding has been provided. Now you can implement required behaviour on client project.